src/bp-core/bp-core-functions.php

```diff
@@ -3125,6 +3125,9 @@ function bp_core_get_suggestions( $args ) {
 		return new WP_Error( 'missing_parameter' );
 	}
 
+	//Remove action for remove search against xprofile fields
+	remove_action( 'bp_user_query_uid_clauses', 'bp_xprofile_bp_user_query_search', 10, 2 );
+
 	$suggestions = new $class( $args );
```

Comment:
@manishpatolia This is correct like to remove the search from the Xprofile Fields but we can't remove the search from the Xprofile `First, Last & Nickname` field. So I would suggest you create a new function that will allow searching in Xprofile `First, Last & Nickname` field.

Reply:
Done as per your suggestion...